src/tests/makefile.all

```diff
@@ -340,33 +346,38 @@ NO_W32_CFLAGS = -D_Windows
   endef
 
 @elifdef WATCOM
-  LDFLAGS = option quiet, map, eliminate, caseexact
+  LDFLAGS = option quiet, map, eliminate
```

Comment:
Why dropping `caseexact`? It's important not to mix symbols with different casing IMHO.

Reply:
I will submit other fixes to watcom make files to fix some compiler and linker illogical options etc.
If you don't specify your own target (not supported by watcom) then you should use standard `system` directive as first. Important options and directives are handled by `system` directive.
After this directive you can modify anything if you need to change default setup by system directive. Now only 16-bit windows target use case insensitive symbols comparision/selection all others are case sensitive.